app/src/main/res/values/strings.xml

```diff
@@ -24,4 +24,23 @@
     <string name="save_button">Save</string>
     <string name="at">At</string>
     <string name="delete_task_button">DELETE TASK</string>
+
+    <!--Reminder time options popup menu on agenda item detail screen -->
+    <string name="ten_minutes_before">10 minutes before</string>
+    <string name="thirty_minutes_before">30 minutes before</string>
```

Comment:
For such strings, you could use placeholder string resources, like this:
`<string name="x_min_before">%d minutes before/>`

And then in your code:
` context.getString(R.string.x_min_before, 10)` will create `10 minutes before`.

That way, you only have one string resource per minute, hour and day.